src/components/views/myDelegations/GetDonations.js

```diff
@@ -0,0 +1,58 @@
+import { paramsForServer } from 'feathers-hooks-common';
```

Comment:
This file and GetDonationsService file are not much meaningful. If it's possible please merge these two in DonationsService.jsx file

Reply:
I changed donationService to donationBlockchainService.